Amplify/Categories/DataStore/Model/Temporal/Time.swift

```diff
@@ -8,46 +8,64 @@
 import Foundation
 
 extension Temporal {
-
-    /// `Time` is an immutable `TemporalSpec` object that represents a time, often viewed
-    /// as hour-minute-second (`HH:mm:ss`). Time can be represented to nanosecond precision.
-    /// For example, the value "13:45:30.123". It can also hold a reference to a TimeZone.
+    /// `Temporal.Time` represents a `Time` with specific allowable formats.
     ///
-    /// As all Temporal types, `Time` relies on the ISO8601 calendar and fixed format.
-    public struct Time: TemporalSpec, TimeUnitOperable {
+    ///  * `.short` => `HH:mm`
+    ///  * `.medium` => `HH:mm:ss`
+    ///  * `.long` => `HH:mm:ss.SSS`
+    ///  * `.full` => `HH:mm:ss.SSSZZZZZ`
+    public struct Time: TemporalSpec {
+        // Inherits documentation from `TemporalSpec`
+        public let foundationDate: Foundation.Date
 
-        public static var iso8601DateComponents: Set<Calendar.Component> {
```

Comment:
Breaking change? I see that it is not used, should we deprecate this and make an internal one for  the init method below?

Reply:
Yes, good catch.

Reply:
Addressed in: [cdace94](https://github.com/aws-amplify/amplify-ios/pull/1760/commits/cdace94592eca4ee9c74d5ac7b52811187b179c1)